routes/web.php

```diff
@@ -65,7 +65,10 @@
         Route::get('notifications/recent', [NotificationController::class, 'getRecentNotifications'])->name('notifications.getNotifications');
         Route::resource('notifications', NotificationController::class);
         Route::get('/my-services/bookings', [EventServiceController::class, 'showBookings'])->name('event-services.bookings');
+        Route::get('/my-confirmed/bookings', [EventServiceController::class, 'showConfirmedBookings'])->name('confirmed.bookings');
```

Comment:
it works but you can make it more conventional by using [modelName].[verb/property]